RxBlocking/BlockingObservable+Operators.swift

```diff
@@ -197,4 +197,64 @@ extension BlockingObservable {
 
         return element
     }
+
+    /**
+     Blocks current thread until sequence terminates.
+     
+     If sequence terminates with error before producing first element, terminating error will be thrown.
+     
+     - parameter predicate: A function to test each source element for a condition.
+     - returns: Returns the only element of an sequence that satisfies the condition in the predicate, and reports an error if there is not exactly one element in the sequence.
+     */
+    public func single(predicate: (E) throws -> Bool) throws -> E? {
```

Comment:
The no-predicate version now uses the predicate version, just added `try` before `single`.